lib/spack/spack/relocate.py

```diff
@@ -232,7 +232,7 @@ def needs_text_relocation(filetype):
     """
     Check whether the given filetype is text that may need relocation.
     """
-    if "link" in filetype:
+    if "link to" in filetype:
```

Comment:
Is there a reason you chose `"link to"` vs. `"symbolic link"`? The latter seems clearer to me

Reply:
I wanted to cover the case of hard links but it turns out file reports hard links as the parent file type.